components/Result/Result.scss

```diff
@@ -0,0 +1,99 @@
+.result {
+  display: none;
+
+  &__wrapper {
+    @include tablet {
+      display: flex;
+      align-items: center;
+      gap: 24px;
+    }
+
+    @include desktop-xl {
+      gap: 32px;
+    }
+  }
+
+  &__cards {
+    @include tablet {
+      margin-left: auto;
+      width: calc(50% - 12px);
+
+      > *:not(:last-child) {
+        margin-bottom: 24px;
+      }
+    }
+
+    @include desktop-xl {
+      width: calc(50% - 20px);
+
+      > *:not(:last-child) {
+        margin-bottom: 32px;
+      }
+    }
+  }
+
+  &__image {
+    img {
+      width: 100%;
+      height: 100%;
+    }
```

Comment:
if you set the width and height of the img tag, then you need to add display: block. Since this tag has display: inline by default. Please read the difference between these properties [link](https://www.freecodecamp.org/news/the-css-display-property-display-none-display-table-inline-block-and-more/)